HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -256,6 +257,7 @@ Eigen::Matrix3d rollPitchYawToRotationMatrix(const Eigen::Array3d &rollPitchYaw,
                     * Eigen::AngleAxisd(rollPitchYaw[0], Eigen::Vector3d::UnitX()))
                 .matrix();
             break;
```

Comment:
Unreachable code. Clang has a warning for this, I expect it to cover this. You should enable it, if it is already I am surprised it does not fire on this. 

The break after return will never be hit, you should delete it.

Reply:
This sample is not tested in CI due to #43, so Clang hasn't had the chance to complain yet.

Reply:
Removed

Reply:
Tip: Since most samples aren't built in CI yet, it's a good idea to enable warnings when building locally.